carousel/contrib/readers.py

```diff
@@ -19,9 +19,11 @@ def copy_model_instance(obj):
     """
     https://djangosnippets.org/snippets/1040/
```

Comment:
snippet doesn't seem to work...
I'm trying to understand where '_meta' comes from right now. And 'get_fields' as well.